Gemfile

```diff
@@ -38,6 +38,7 @@ group :test do
   gem "capybara", ">= 2.15"
   gem "database_cleaner-active_record"
   gem "faker"
+  gem "rails-controller-testing"
```

Comment:
thêm gem này để làm gì e?

Reply:
e dùng gem này để  assigns các biến instance ở trong controller 
ví dụ action destroy có @product.destroy 

bên test: 
 it "destroy product" do
   expect(assigns(:product).destroyed?).to eq true
 end